api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedRowError.java

```diff
@@ -35,6 +36,11 @@
     @Column(name = "error_type")
     private ValidationCategory errorType;
 
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "error_level")
```

Comment:
This column will need to be added to the DDL as well right?

Reply:
StagedRowError is in a weird state. It's currently used as a regular model not JPA(never persited)
I have left the JPA definition in case we have  to save it for better performances once we get real(bigger) spreadSheet.

But I m happy to add the field to the DDL

Reply:
Heh yeah weird state. Yeah,. add the field to DDL: no point having it lying around if it's going to be in an unusable state when we need it.